HRIS.Services/Services/ChartService.cs

```diff
@@ -97,16 +97,6 @@ public async Task<List<ChartDto>> GetAllCharts()
 
             var labels = roleDictionaries.Values.SelectMany(dict => dict.Keys).Distinct().OrderBy(label => label).ToList();
             
-            foreach (var dictionary in roleDictionaries.Values)
```

Comment:
So I'm guessing you tested and are sure we don't need it

Reply:
Yes, I tested the front-end functionality for the charts. I added, deleted, edited the charts and everything works perfectly.